examples/automated-summary-report/main.bal

```diff
@@ -0,0 +1,70 @@
+import ballerina/io;
+import ballerina/log;
+import ballerinax/slack;
+
+// Define the Slack API token
+configurable string token = ?;
+
+// Define a record type to hold channel information
+type ChannelType record {
+    string id;
+};
+
+// Define a record type to hold the response structure of the channels list
```

Comment:
Please update the all the record related comments into docs using the below pattern.
```suggestion
# Holds the response structure of the channels list.
```